src/graph/executor/query/DataCollectExecutor.cpp

```diff
@@ -64,8 +64,6 @@ folly::Future<Status> DataCollectExecutor::doCollect() {
 Status DataCollectExecutor::collectSubgraph(const std::vector<std::string>& vars) {
   DataSet ds;
   ds.colNames = std::move(colNames_);
-  // the subgraph not need duplicate vertices or edges, so dedup here directly
-  std::unordered_set<Value> uniqueVids;
```

Comment:
We don't need deduplication now ?

Reply:
subgraphExecutor guaranteed that vid is unique